ERCS/erc-ppo-authorization-object.md

```diff
@@ -0,0 +1,217 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
eip: 8103
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.